bootcd/genesis.ks.template

```diff
@@ -220,6 +220,8 @@ echo ">>>>> Removing bloat from /usr/share"
 rm -rf /usr/share/doc
 rm -rf /usr/share/icons
 
+sed -e 's|ACTIVE_CONSOLES.*|ACTIVE_CONSOLES="/dev/tty1 /dev/ttyS0 /dev/ttyS1"|' -i /etc/sysconfig/init
```

Comment:
I'm fairly certain the line as is doesn't contain a `"` mark after the equal sign which is needed for it to properly capture this information which has to be separated by spaces.
IIRC (it will take a few minutes for me to rebuild and check) I'm going from:
`ACTIVE_CONSOLES=/dev/tty1`
to
`ACTIVE_CONSOLES="/dev/tty[1-6] /dev/ttyS[0-1]"`
(which I just tested and does appear to work so I'll be updating the PR shortly)

"Appending" could be implemented as the slightly more complicated:
1. remove any `"` after the `=` and at the end of the line if they are there.
2. append ` /dev/ttyS[0-1]`
3. put back in the necessary quotation marks

I'm happy to spend a few minutes writing that if it would be preferable, though if the current value is only `/dev/tty1` then explicitly setting it to `/dev/tty[1-6]` could be preferable based on your original feedback. Thoughts?